scripts/repair/repair_missing_comments.py

```diff
@@ -0,0 +1,35 @@
+#
+# Usage:
+#    To repair comment ids "0" - "5":
+#       sudo reddit-run scripts/repair/repair_missing_comments.py -c 'repair_comment_range("0", "5")'
+#
+
+from r2.lib import amqp
+from r2.lib.db.thing import NotFound
+from r2.models.link import Comment
+
+
+def repair_comment_range(comment_start_id, comment_end_id):
```

Comment:
Should we set these to be kwargs with a default value of `None` so that if either of them are omitted then it will span the maximum value on either end of the available range? (Is that even possible?)

Reply:
I can look into figuring out what that range is...